docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLpNorm.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Samples.Dynamic
+{
+    class NormalizeLpNorm
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+            var samples = new List<DataPoint>()
+            {
+                new DataPoint(){ Features = new float[4] { 1, 1, 0, 0} },
+                new DataPoint(){ Features = new float[4] { 2, 2, 0, 0} },
+                new DataPoint(){ Features = new float[4] { 1, 0, 1, 0} },
+                new DataPoint(){ Features = new float[4] { 0, 1, 0, 1} }
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+            // NormalizeLpNorm normalize rows individually by rescaling them to unit norm.
+            // Performs the following operaion on a row X:  Y = (X - M) / D where M is mean, and D is selected norm.
```

Comment:
What is the selected norm? Is it norm of the feature vector in a row being processed? Also, what are the shapes of X, Y, M, and D?

Reply:
`mean` -> `mean vector` 
`D is selected norm` -> `D is calculated value of selected norm parameter` 
Does that sound better?

---
In reply to: [273152443](https://github.com/dotnet/machinelearning/pull/3232#discussion_r273152443) [](ancestors = 273152443)

Reply:
Norm on what? A column? A row? Is it a scalar? In tensor computation, norm operation can produce another tensor. 

Say, if I have rows, x_1, x_2, x_3. Is D=1/3 (x_1 + x_2 + x_3) true? Or D=ReduceSum(x_i) for the x subscripted by i? In addition, is D=||x_i||_2 for the x subscripted by i?

---
In reply to: [273153905](https://github.com/dotnet/machinelearning/pull/3232#discussion_r273153905) [](ancestors = 273153905,273152443)

Reply:
Same here. Please move the final answer to what NormalizeLpNorm does' to the <remarks> section for the estimator.

---
In reply to: [273181155](https://github.com/dotnet/machinelearning/pull/3232#discussion_r273181155) [](ancestors = 273181155,273153905,273152443)